src/commentbot/CommentBot.jl

```diff
@@ -0,0 +1,252 @@
+module CommentBot
+
+using Sockets
+using GitHub
+using HTTP
+using Distributed
+using Base64
+using Pkg
+using Logging
+using Dates
+using JSON
+using MbedTLS
+
+import Pkg: TOML
+import ..Registrator: post_on_slack_channel, pull_request_contents
+import ..RegEdit: RegBranch, RegisterParams
+import Base: string
+using ..Messaging
+
+include("trigger_types.jl")
+include("parse_comment.jl")
+include("github_utils.jl")
+include("verify_projectfile.jl")
+include("param_types.jl")
+include("approval.jl")
+
+include("../management.jl")
+
+const event_queue = Channel{RequestParams}(1024)
+const config = Dict{String,Any}()
+const httpsock = Ref{Sockets.TCPServer}()
+
+function print_entry_log(rp::RequestParams{PullRequestTrigger})
+    @info("Creating registration pull request for $(rp.reponame) PR: $(rp.trigger_src.prid)")
+end
+
+function print_entry_log(rp::RequestParams{CommitCommentTrigger})
+    @info("Creating registration pull request for $(rp.reponame) sha: `$(get_comment_commit_id(rp.evt))`")
+end
+
+function print_entry_log(rp::RequestParams{IssueTrigger})
+    @info("Creating registration pull request for $(rp.reponame) branch: `$(rp.trigger_src.branch)`")
+end
```

Comment:
Should the above log messages use kwarg logging?
Similar ones below use kwarg logging.

Possible advantage is easier to computationally process.

Reply:
Fixed in f44f440fd04a4b6a458e33147a83e4bad92dfb41

Don't know why it doesn't show as outdated.